src/Http/Controllers/Actions/FetchMany.php

```diff
@@ -61,6 +61,6 @@ public function index(Route $route, StoreContract $store)
             $response = $this->searched($data, $request);
         }
 
-        return $response ?: new DataResponse($data);
+        return $response ?: (new DataResponse($data))->withRequest($request);
```

Comment:
Use the static `make` method now that we need to chain off this.

```php
DataResponse::make($data)->withQueryParameters($request)
```

Reply:
Done.